compiler/rustc_trait_selection/src/traits/error_reporting/mod.rs

```diff
@@ -2265,7 +2265,7 @@ impl<'a, 'tcx> InferCtxtPrivExt<'a, 'tcx> for InferCtxt<'a, 'tcx> {
                             };
                             let mut suggestions = vec![(
                                 trait_path_segment.ident.span.shrink_to_lo(),
```

Comment:
this should probably be `path.shrink_to_lo()` :thinking: not completely sure whether that's correct.